### PR TITLE
added missing dictionaries which were sliently ignored due to root bug

### PR DESCRIPTION
**Pull request by smuzaffar**
this should fix the root6 relvals

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: DataFormats/CTPPSReco/src/classes.h [hunk 1]
```diff
@@ -35,5 +35,8 @@ namespace DataFormats_CTPPSReco {
     edm::Wrapper<edm::DetSetVector<TotemRPLocalTrack>> w_dsv_ft;
     edm::DetSetVector<TotemRPLocalTrack::FittedRecHit> dsv_ft_frh;
     edm::Wrapper<edm::DetSetVector<TotemRPLocalTrack::FittedRecHit>> w_dsv_ft_frh;
+   
+    std::vector<edm::DetSet<TotemRPLocalTrack::FittedRecHit> > v_ds_ft_frh;
```
Comment (Dr15Jones) [inline DataFormats/CTPPSReco/src/classes.h]: I believe with ROOT 6 it is no longer necessary to add templated classes to the classes.h.

Comment (smuzaffar) [inline DataFormats/CTPPSReco/src/classes.h]: so we can remove these from classes.h but keep them in classes_xml.def file?

Comment (Dr15Jones) [inline DataFormats/CTPPSReco/src/classes.h]: I believe that is correct. There is no need to do that here, although we should eventually have a cleanup campaign to fix all classes.h files.

**Comment by smuzaffar** [comment]
merging it in root6 IBs to see what else is broken

**Comment by smuzaffar** [comment]
backport done
Successfully backported PR #16832 as #16893 for branch CMSSW_9_0_X

**Pull request merged**